electrumx/server/session.py

```diff
@@ -142,6 +143,39 @@ def __init__(self, env, db, bp, daemon, mempool, shutdown_event):
         LocalRPC.request_handlers = {cmd: getattr(self, 'rpc_' + cmd)
                                      for cmd in cmds}
 
+    async def _start_ws_server(self, *args, **kw_args):
+        instance = ElectrumX(self, self.db, self.mempool, self.peer_mgr, 'TCP')
+
+        @asyncio.coroutine
```

Comment:
The decorator is not necessary; it appears nowhere else in ElectrumX.

Reply:
Removed